.pre-commit-config.yaml

```diff
@@ -1,5 +1,10 @@
 default_language_version:
   python: python3
+default_install_hook_types:
+  - pre-commit
+  - pre-push
+default_stages:
+  - pre-commit
```

Comment:
By default, installing `pre-commit` only install `pre-commit` hooks. Now that there is a `pre-push` hook we need to install those too.

However all other hooks should only run on `pre-commit`, no need to check everything again on `push`, so set `default-stages` to `pre-commit` only.